war/src/main/less/modules/table.less

```diff
@@ -119,13 +129,17 @@
     height: 26px;
   }
 
-  //& > thead > tr:hover, & > tbody > tr:hover {
-  //  .jenkins-table__button::before, .sortheader::before {
-  //    opacity: 0.03;
-  //  }
-  //}
+  &__button {
+    display: flex;
+  }
+
+  &__link, .sortheader {
+    display: inline-flex;
+  }
```

Comment:
This `inline-flex` may have caused [JENKINS-68205](https://issues.jenkins.io/browse/JENKINS-68205).